### PR TITLE
chore: prepare release 2024-07-26

### PR DESCRIPTION
**Pull request by shortcuts**
## Summary

This PR has been created using the `apic release` script. Once merged, the clients will try to release their new version if their version has changed.

## Version Changes

- csharp: 7.0.0-beta.10 -> **`prerelease` _(e.g. 7.0.0-beta.11)_**
- dart: 1.18.0 -> **`minor` _(e.g. 1.19.0)_**
- go: 4.0.0-beta.24 -> **`prerelease` _(e.g. 4.0.0-beta.25)_**
- java: 4.0.0-beta.41 -> **`prerelease` _(e.g. 4.0.0-beta.42)_**
- javascript: 5.0.0-beta.12 -> **`prerelease` _(e.g. 5.0.0-beta.13)_**
- kotlin: 3.0.0-beta.36 -> **`prerelease` _(e.g. 3.0.0-beta.37)_**
- php: 4.0.0-beta.10 -> **`prerelease` _(e.g. 4.0.0-beta.11)_**
- python: 4.0.0b25 -> **`minor` _(e.g. 4.0.0b26)_**
- ruby: 3.0.0.beta.10 -> **`minor` _(e.g. 3.0.0.beta.11)_**
- scala: 2.0.0-beta.10 -> **`prerelease` _(e.g. 2.0.0-beta.11)_**
- swift: 9.0.0-beta.11 -> **`prerelease` _(e.g. 9.0.0-beta.12)_**

### Skipped Commits

<p>It doesn't mean these commits are being excluded from the release. It means they're not taken into account when the release process figured out the next version number, and updated the changelog.</p>

<details>
  <summary>
    <i>Commits without language scope:</i>
  </summary>

  - chore: generated code for commit 4455b77a681309bf173e0b0fd6a0894de9de68e9. [skip ci]
- chore: generated code for commit ed9e145550ef3299224ebfce7e766bdf4c79a5ce. [skip ci]
- chore: generated code for commit 200f07e9cfc0b6dce090496a41e67247f38e0e10. [skip ci]
- chore: generated code for commit 81c15cf5bcf0931f3ab8b0204ca06c5a004f7534. [skip ci]
- chore: generated code for commit be060a0600b6e32a69604e6ba45552fca9a2ebdc. [skip ci]
- chore: generated code for commit fe6f5edabc9f86740e09ed1984119b623858fc3e. [skip ci]
- chore: generated code for commit e3c6b84017da7a37eec083ce95e88862850028d0. [skip ci]
- chore: generated code for commit e974ffcb9ef0fc20e553f420dcdfab0347e5221d. [skip ci]
- chore: generated code for commit b0fa5a91521ad3b7fb707da13cbe514f074050e3. [skip ci]
- chore: use registered java version (#3409)
- chore: generated code for commit 0e13b695f1cc88aea7103ac5664b678e4efdef41. [skip ci]
- chore: generated code for commit 23fd55c51cf2cce67887e8241e2fced1b7259d41. [skip ci]
- chore: generated code for commit 328f911e318343ace3d145536be539a714d7164a. [skip ci]
- chore: generated code for commit 82562cc5a30fe2d51836aebd797b8cfcd7cf732a. [skip ci]
</details>

<details>
  <summary>
    <i>Commits with unknown language scope:</i>
  </summary>

  - chore(playgroud): add a build playground step (#3420)
- fix(scripts): mention generated is spread msg commit (#3421)
- chore(scripts): improve generation commit message (#3419)
- fix(cts): assert the error of chunkedBatch helpers (#3418)
- chore(cts): assert the retry strategy error (#3405)
- chore(cts): report benchmark results (#3406)
- chore(CTS): benchmark the search method (#3328)
- chore(deps): dependencies 2024-07-22 (#3389)
</details>

### CODE REVIEW

File: scripts/ci/codegen/text.ts [hunk 1]
```diff
@@ -12,5 +12,5 @@ export default {
 };
 
 export function isGeneratedCommit(text: string): boolean {
-  return text.endsWith(commitEndMessage);
+  return text.includes(commitEndMessage);
```
Comment (shortcuts) [inline scripts/ci/codegen/text.ts:15]: ok this is enough it seems

**Review by millotp — APPROVED** [review]

**Comment by shortcuts** [comment]
let's see if it breaks the release or not, shouldn't be!

**Comment by shortcuts** [comment]
@millotp you forgot to put the super enthusiastic YES message :/

**Comment by millotp** [comment]
YES !!!! 🥳 🎊

**Pull request merged**